### PR TITLE
added svm task

### PR DESCRIPTION
**Pull request by Nafisay**

### CODE REVIEW

File: ML/svm.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## What you need to do
+
+### Part I
+* Familiarize yourself with basics of Machine Learning.
+* Explore Support Virtual Machines - its different types, practical use-cases and algorithmic analysis. You are also supposed to explore the sci-kit learn library for SVMs.
+* You need to make a presentation explaining what is SVM and how is it different from other popular Machine Learning algorithms (ex. kNNs, etc.) your thought process and how / why you choose a certain tool while solving this problem.
+
+### Part II
+* Pick a dataset (from ex. [Kaggle](https://www.kaggle.com/datasets)).
+* Write a code, using sci-kit learn (svm), to perform a non-linear classification. However, keep in mind that you will be required to explain the working of the algorithm you have used.
```
Comment (somya-kr) [inline ML/svm.md:22]: How about writing svm from scratch (using numpy and pandas) ?

Comment (Nafisay) [reply ML/svm.md]: done

**Review by somya-kr — APPROVED** [review]
LGTM

**Comment by proffapt** [comment]
I guess it's ready to be merged, if one of the executives of the same field has approved the PR already. And if you want someone else to have a look at it as well, then directly request them.

cc: @harshkhandeparkar @somya-kr @lurkingryuu

**Review by harshkhandeparkar — APPROVED** [review]

**Comment by harshkhandeparkar** [comment]
merging this then.

**Pull request merged**